README.md

```diff
@@ -38,3 +37,25 @@ $ sentry-kube --help
 * `SENTRY_KUBE_KUBECTL_VERSION`: Set `SENTRY_KUBE_KUBECTL_VERSION=1.22.17` to configure the kubectl version to use
 * `SENTRY_KUBE_NO_CONTEXT`: Set `SENTRY_KUBE_NO_CONTEXT=1` to skip checking for a functional kube context
 * `SENTRY_KUBE_ROOT`: Sets the workspace root. It defaults to the git root directory.
+
+## How to use sentry-infra-tools in editable mode (for development) in another environment
+
+Lets assume you have a local working copy of sentry-infra-tools in
+`~/sentry-infra-tools`. Lets assume that you made some change
+in your local copy of sentry-infra-tools. But you would like to validate
+the change in a different virtual environment. Here is how you can do it:
+
+1. Remove the existing sentry-infra-tools package from the environment
+   where you want to test it out.
+
+```shell
+pip uninstall sentry-infra-tools
+```
+
+2. Install the local working copy of sentry-infra-tools in editable mode.
+
+```shell
+pip install -e ~/sentry-infra-tools
```

Comment:
Wouldn't it work if we just asked people to make a change to requirements.in/txt and reference the local copy instead? 
With this approach refreshing dependencies in the ops repo (devenv sync) would remove the local copy and go back to the public one.

Reply:
I added the additional instructions on how to work with requirements.txt file. Since sentry-infra-tools is public, I want the instructions to be generic and not catered only towards the ops repo (even though it is setup like that today).